README.md

```diff
@@ -177,6 +170,22 @@ root@9feae5e5e001:/src/conjur/cucumber/api# cucumber
 root@9feae5e5e001:/src/conjur# cucumber -r cucumber/api/features/support -r cucumber/api/features/step_definitions cucumber/api/features/resource_list.feature
 ```
 
+## Documentation site
+
+This repository also contains the entire source code for the [Conjur
+documentation website][docs]. For instructions on how to work on the
+site locally, visit the [docs README][docs-readme].
+
+Or in brief:
+
+```sh-session
+$ docker-compose up -d docs
+$ open localhost:4000
+```
+
+[docs]: https://try.conjur.org "Conjur website"
```

Comment:
Oh neat, I didn't know you could do this.

Reply:
:tipping_hand_man: